tests/console/yast2_lan_device_settings.pm

```diff
@@ -24,8 +24,10 @@ use version_utils qw(is_sle is_leap is_tumbleweed);
 use y2lan_restart_common;
 
 sub run {
-    my $self      = shift;
-    my $static_ip = "192.168.1.119";
+    my $self               = shift;
+    my $static_ip          = "192.168.1.119";
+    my $static_hostname    = 'testhost';
```

Comment:
Why use `testhost` instead of `localhost` now?

Reply:
I agree with YaST2 dev about it. Not necessary from the functional point though.

Reply:
As a solution which will resolve this discussion, we could move that to the variable. Testing something different to `localhost` would be beneficial.

Reply:
I'm fine with `testhost` as well, just wondering why the change is part of this PR.

Reply:
Legit question. Just short-circuiting the process since it is a tiny change. Therefore, I have asked more people for review.